linkerd-viz

```diff
@@ -56,8 +57,14 @@ curl -vX PUT -d"{\"theme\": \"dark\"}" -H "Content-Type: application/json" http:
 # init prometheus, exec as pid 1
 #
 
+# set scrape and evaluation intervals
+PROMETHEUS_CONF=/etc/prometheus/prometheus-$PLATFORM.yml
+
+sed -i.tmp "s@scrape_interval:     1m@scrape_interval:     $SCRAPE_INTERVAL@" $PROMETHEUS_CONF
+sed -i.tmp "s@evaluation_interval: 1m@evaluation_interval: $SCRAPE_INTERVAL@" $PROMETHEUS_CONF
```

Comment:
TIOLI: These regexes seem a bit brittle to me. Maybe consider just replacing everything after the `:` with the updated scrape interval. Something like:

```
sed -i "" "s@scrape_interval:.*@scrape_interval: $SCRAPE_INTERVAL@" $PROMETHEUS_CONF
sed -i "" "s@ evaluation_interval:.*@ evaluation_interval: $SCRAPE_INTERVAL@" $PROMETHEUS_CONF
```